### DISCUSSION TITLE
KeepLargestConnectedComponent: Is input image or label?

### DISCUSSION THREAD

**Discussion by Joeycho** [Project-MONAI/MONAI#5305] [General]
https://github.com/Project-MONAI/MONAI/blob/8087dc58ea6803099a0466abf62f420d7cdc9d2e/monai/transforms/post/array.py#L354

With the given matrix format examples, I understood input as label, rather than image.

Is there any reason behind this? I think with "label", the meaning is clearer for users if I did not misunderstand.

**Comment by KumoLiu** [comment]
Hi @Joeycho, the `img` here is just the generalized input image, also to keep it consistent with other transforms. The docstring also explains what data should be entered in detail. Thanks for your advice!

**Reply by Joeycho** [reply]
I see. I just checked other transforms, such as LabelFilter, Fillhole, etc. Indeed. I think people can understand what the given transform does with docstring in class.
